05-oop.md

```diff
@@ -1,40 +1,40 @@
-# 4 Pillars of OOP
-- Abstraction, polymorphism, inheritance & encapsulation.
```

Comment:
It's removed because I don't see it as a key point of OOP. It's a part of any form of programming, regardless of paradigm. You could argue that the `abstract` keyword is part of OOP in Java, but even that doesn't mean much IMO.

Also, I believe I mentioned only three 'pillars' when I went over this as that's what I was taught - not that that makes it correct or anything. :)